### ISSUE TITLE
[Login page] Components wiggle down screen when an error is shown

### ISSUE THREAD

**Issue by richard-cox** [rancher/dashboard#4202]
- ~Attempt to log in via a local user with junk credentials, the error message is shown at the top and all components underneath wiggle down screen~ This requires more of a redesign of the whole page
- This problem is made worse when there's a status message. For example log out (url is `/auth/login?logged-out`) and green status text is shown, on log in attempt this is removed which causes a wiggle... then the error message causes a re-wiggle.
- In summary, log in screen needs de-wiggling.

**Comment by richard-cox** [comment]
#4740 resolves the wiggle on log in

**Comment by nwmac** [comment]
@richard-cox 4740 doesn't look like the right issue - is there another issue for this? or a PR that fixes it?

**Comment by richard-cox** [comment]
@nwmac Sorry, have updated the comment, meant #4722. That solves the wiggle when there's an existing error message and user tries to log in.

**Comment by nickwsuse** [comment]
`Verified on v2.7-head 216bb7f`

I tried a few different scenarios and if there's an existing message (error or otherwise) on the login screen, if another appears it simply replaces the existing one with no wiggling.
